docs/developer/api-reference/mutations/checkout-payment-complete.mdx

```diff
@@ -0,0 +1,41 @@
+---
+id: checkout-payment-complete
+title: checkoutPaymentComplete
+---
+
+Complete an individual payment as part of the checkout. This mutation does not create the order and any webhooks related to this payment will neither create the order.
+In case an additional confirmation step such as 3D secure is required confirmationNeeded flag will be set to True and no authorization will be held until the payment is confirmed with the second call of this mutation.
```

Comment:
I would mention here, that confirmationData is also be returned in the response, it will have the details required to confirm a payment on frontend side

Reply:
Done